<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1800</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
background-color:white;
border-style: solid;

border-width: 1px;
border-radius: 7px;
}
QMainWindow{
	background-color: #37474f;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="display_image">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>900</width>
      <height>720</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
border-color: #00838f ;
border-width: 3px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="my_qlabel" name="display_image_2">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>70</y>
      <width>900</width>
      <height>720</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
border-color: #00838f ;
border-width: 3px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="startVideo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>930</y>
      <width>131</width>
      <height>54</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#eeeeee;
color: black;
border-color: white;</string>
    </property>
    <property name="text">
     <string>Start Video</string>
    </property>
   </widget>
   <widget class="QPushButton" name="joystick_status">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>850</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: #64dd17; 

}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="atMega_status">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>850</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: #c62828;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gradient">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>860</y>
      <width>61</width>
      <height>57</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: qlineargradient(spread:pad, x1:0, y1:0.784, x2:0, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(55, 71, 76, 255));

border-width: 1px;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gradient_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>860</y>
      <width>61</width>
      <height>57</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: qlineargradient(spread:pad, x1:0, y1:0.784, x2:0, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(55, 71, 79, 255));

border-width: 1px;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="error_video">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0,0,0,0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gradient_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>930</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: qlineargradient(spread:pad, x1:0, y1:0.784, x2:0, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(55, 71, 76, 255));

border-width: 1px;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="auto_drive">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>850</y>
      <width>141</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: #00838f ; 

}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="shapes_recognize">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>850</y>
      <width>131</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: #00838f ; 

}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gradient_4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>850</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: qlineargradient(spread:pad, x1:0, y1:0.784, x2:0, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(55, 71, 76, 255));

border-width: 1px;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gradient_5">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>850</y>
      <width>131</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: qlineargradient(spread:pad, x1:0, y1:0.784, x2:0, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(55, 71, 76, 255));

border-width: 1px;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gradient_6">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>850</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: qlineargradient(spread:pad, x1:0, y1:0.784, x2:0, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(55, 71, 76, 255));

border-width: 1px;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="home">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>850</y>
      <width>141</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: #00838f ; 

}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="trackbar_circle">
    <property name="geometry">
     <rect>
      <x>1570</x>
      <y>20</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="maximum">
     <number>400</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberTrackbar">
    <property name="geometry">
     <rect>
      <x>1640</x>
      <y>0</y>
      <width>64</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value" stdset="0">
     <double>0.000000000000000</double>
    </property>
    <property name="intValue" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="logoLabel">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>-70</y>
      <width>641</width>
      <height>291</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="cannon_measure">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>850</y>
      <width>141</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: #00838f ; 

}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gradient_7">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>850</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: qlineargradient(spread:pad, x1:0, y1:0.784, x2:0, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(55, 71, 76, 255));

border-width: 1px;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gradient_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>50</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: qlineargradient(spread:pad, x1:0, y1:0.784, x2:0, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(55, 71, 76, 255));

border-width: 1px;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="term_sensor">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: #00838f ; 

}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="console">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>850</y>
      <width>521</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
border-color: #00838f ;
border-width: 1px;</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600; color:#204a87;&quot;&gt;-- PoliTOcean Carassius v2.0:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="home_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: #00838f ; 

}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gradient_9">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>50</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: qlineargradient(spread:pad, x1:0, y1:0.784, x2:0, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(55, 71, 76, 255));

border-width: 1px;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="measure_button">
    <property name="geometry">
     <rect>
      <x>1720</x>
      <y>690</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: #00838f ; 

}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gradient_10">
    <property name="geometry">
     <rect>
      <x>1720</x>
      <y>690</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: qlineargradient(spread:pad, x1:0, y1:0.784, x2:0, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(55, 71, 76, 255));

border-width: 1px;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblMouse_unit">
    <property name="geometry">
     <rect>
      <x>1310</x>
      <y>10</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Unit: </string>
    </property>
   </widget>
   <widget class="QPushButton" name="home_4">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>10</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: #00838f ; 

}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblMouse_R1">
    <property name="geometry">
     <rect>
      <x>1310</x>
      <y>30</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>R1: </string>
    </property>
   </widget>
   <widget class="QLabel" name="lblMouse_L">
    <property name="geometry">
     <rect>
      <x>1430</x>
      <y>10</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>L:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblMouse_R2">
    <property name="geometry">
     <rect>
      <x>1430</x>
      <y>30</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>R2: </string>
    </property>
   </widget>
   <widget class="QCheckBox" name="debugCheck">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Debug</string>
    </property>
   </widget>
   <zorder>display_image</zorder>
   <zorder>display_image_2</zorder>
   <zorder>gradient</zorder>
   <zorder>joystick_status</zorder>
   <zorder>gradient_2</zorder>
   <zorder>atMega_status</zorder>
   <zorder>error_video</zorder>
   <zorder>gradient_3</zorder>
   <zorder>startVideo</zorder>
   <zorder>gradient_4</zorder>
   <zorder>gradient_5</zorder>
   <zorder>shapes_recognize</zorder>
   <zorder>auto_drive</zorder>
   <zorder>gradient_6</zorder>
   <zorder>home</zorder>
   <zorder>lcdNumberTrackbar</zorder>
   <zorder>logoLabel</zorder>
   <zorder>gradient_7</zorder>
   <zorder>cannon_measure</zorder>
   <zorder>gradient_8</zorder>
   <zorder>term_sensor</zorder>
   <zorder>trackbar_circle</zorder>
   <zorder>gradient_9</zorder>
   <zorder>home_3</zorder>
   <zorder>console</zorder>
   <zorder>gradient_10</zorder>
   <zorder>measure_button</zorder>
   <zorder>lblMouse_unit</zorder>
   <zorder>home_4</zorder>
   <zorder>lblMouse_R1</zorder>
   <zorder>lblMouse_L</zorder>
   <zorder>lblMouse_R2</zorder>
   <zorder>debugCheck</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>my_qlabel</class>
   <extends>QLabel</extends>
   <header location="global">my_qlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
